ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -65,13 +70,14 @@ protected void onScrollChanged(int x, int y, int oldX, int oldY) {
 
   @Override
   public boolean onInterceptTouchEvent(MotionEvent ev) {
-    if (super.onInterceptTouchEvent(ev)) {
-      NativeGestureUtil.notifyNativeGestureStarted(this, ev);
-      ReactScrollViewHelper.emitScrollBeginDragEvent(this);
-      mDragging = true;
-      return true;
+    if (mScrollEnabled) {
```

Comment:
The two nested ifs should be on one line.
`if (mScrollEnabled && super.onInterceptTouchEvent(ev))`